File: app/src/main/java/com/yingyang/extractaudiovideo/MyApplication.java
package com.yingyang.extractaudiovideo;

import android.app.Application;
import android.content.Context;

import androidx.multidex.MultiDex;

public class MyApplication extends Application {

    @Override
    protected void attachBaseContext(Context base) {
        super.attachBaseContext(base);
        MultiDex.install(this);
    }

    @Override
    public void onCreate() {
        super.onCreate();
        //腾讯Bugly初始化
//        CrashReport.initCrashReport(getApplicationContext(), getString(R.string.bugly_app_id), false);
    }
}

File: app/src/main/java/com/yingyang/extractaudiovideo/MyApplication.kt
package com.yingyang.extractaudiovideo

import android.content.Context
import androidx.multidex.MultiDex
import com.yingyangfly.baselib.BaseApplication

class MyApplication : BaseApplication() {

    override fun attachBaseContext(base: Context?) {
        super.attachBaseContext(base)
        //如果需要使用MultiDex，需要在此处调用。
        MultiDex.install(this)
    }
}